platform/mikro-e/dev/motion-click.h

```diff
@@ -44,43 +44,46 @@
 #define MOTION_SENSOR_PIN       0
 
 /* To initialize interrupt on Motion detected */
-#define MOTION_SENSOR_IRQ_INIT() __MOTION_SENSOR_IRQ_INIT()
-#define __MOTION_SENSOR_IRQ_INIT()                                      \
-  do {                                                                  \
-    CNCONDSET = _CNCOND_ON_MASK;                                        \
-    IEC1SET = _IEC1_CNDIE_MASK;                                         \
-    IFS1CLR = _IFS1_CNDIF_MASK;                                         \
-    IPC8SET = (6 << _IPC8_CNIP_POSITION) | (0 << _IPC8_CNIS_POSITION);  \
-    (void)PORTD;                                                        \
+#define MOTION_SENSOR_IRQ_INIT(port) __MOTION_SENSOR_IRQ_INIT(port)
```

Comment:
port and pin need not be arguments for these macros as these are going to be MOTION_SENSOR_PORT and MOTION_SENSOR_PIN always